test/testastutils.cpp

```diff
@@ -72,6 +73,38 @@ class TestAstUtils : public TestFixture {
         ASSERT_EQUALS(true, findLambdaEndToken("[](void) constexpr -> const * const* int { return x }"));
     }
 
+    bool findLambdaStartToken(const char code[]) {
+        Settings settings;
+        Tokenizer tokenizer(&settings, this);
+        std::istringstream istr(code);
+        tokenizer.tokenize(istr, "test.cpp");
+        const Token * const tokStart = ::findLambdaStartToken(tokenizer.list.back());
+        return tokStart && tokStart == tokenizer.list.front();
+    }
+
+    void findLambdaStartToken() {
+        ASSERT(nullptr == ::findLambdaStartToken(nullptr));
+        ASSERT_EQUALS(false, findLambdaStartToken("void f() { }"));
+        ASSERT_EQUALS(true, findLambdaStartToken("[]{ }"));
+        ASSERT_EQUALS(true, findLambdaStartToken("[]{ return 0 }"));
```

Comment:
you have missed a semicolon in a few of these tests

Reply:
Yea, these were copied from the other lambda tests. I can update those too.